src/manjiri/arrayEx/ArrayEx14.java

```diff
@@ -0,0 +1,59 @@
+package manjiri.arrayEx;
+
+import java.util.Arrays;
+
+public class ArrayEx14 {
+	
+/*****************Print all the repeated word in given string.****************/
+//	String str = "Maulik Puja Hindavi Shubham Hindavi Shubham Titiksha Swati";
+//	output : Hindavi 
+//	         Shubham 
+/*****************************************************************************/
+
+	void PrintRepeatedWords(String input) {
+		String[] stringArr = input.split(" ");
+
+		System.out.println("Input array is: " + Arrays.toString(stringArr));
+		System.out.println("Duplicate names are: ");
+		for (int index = 0; index < stringArr.length; index++) {
+			for (int i = index + 1; i < stringArr.length; i++) {
+				if (stringArr[index].equals(stringArr[i]) && (index != i)) {
+					System.out.println(stringArr[index]);
+				}
+			}
+		}
+	}
+	
+/******************Print all the non repeated word in given string.******************/
+//	String str = "Maulik Puja Hindavi Shubham Hindavi Shubham Titiksha Swati";
+//	output : Maulik
+//			 Puja
+//			 Titiksha
+//			 Swati
+/***********************************************************************************/
+
+	void PrintNonRepeatedWords(String input) {
+		String[] stringArr = input.split(" ");
+		System.out.println("Input array is: " + Arrays.toString(stringArr));
+		System.out.println("Non Repeated names are: ");
+		for (int index = 0; index < stringArr.length; index++) {
+			int count = 0;
+			for (int i = 0; i < stringArr.length; i++) {
+				if (stringArr[index].equals(stringArr[i]) && (index != i)) {
+					count++;
```

Comment:
if count++ then simply break the loop, we don;t require to check that word is repeated or not